### PR TITLE
v1.2.2 release

### PR DESCRIPTION
**Pull request by bryce-turner**
This is primarily a nuisance fix release that adjusts memory usage, chunking windows, and task defaults. We've additionally added support for lane level metrics to upload to LIMS, this is currently in use for duplicate metrics/stats, but it should work for any other metric that is produced at the lane level, simply need to add the field to LIMS. 

Bugfixes:
* Set snpSniffer to default to false instead of true, this means that if all tasks are disabled or missing in the config we will only run the default alignment with bwa
* Fixed input/output for bcftools annotate - previously the annotation databases were considered as outputs, not inputs
* Shifted deepvariants dynamic memory allocation to be ~16G higher per bin, which is less prone to OOM kills in our testing
* Adjusted constitutional manta to place output files in the results_dir a bit earlier, which is better for downstream processing
* Added prepare_tumor_only resets to the tumor only variant callers that did not have this set already
* Omitting a denylist region within chunk4 of the calling intervals, this region is prone to significant walltime runouts #65
* Bumped to discordant_loci_extractor v0.1.6 to fix errors when input volume/data is low

Improvements:
* Configured tumor_only sigprofiler to use a more strictly filtered vcf, which yields comparable results to the somatic sigprofiler output
* Updated stats2lims to handle lane level stats #55

### CODE REVIEW

File: required_scripts/uploadStats2Lims.py
Comment (PedalheadPHX) [inline required_scripts/uploadStats2Lims.py:1]: Why was there so any updates

**Review by PedalheadPHX — COMMENTED** [review]
Will wait for final updates before pulling

File: required_scripts/uploadStats2Lims.py
Comment (bryce-turner) [reply required_scripts/uploadStats2Lims.py:1]: Mostly empty space changes from linting. Though I also updated the variables to be more informative based on usage. Lines specifically for lane level uploads are 441-491.

**Comment by bryce-turner** [comment]
Pushed additional changes for handling a numberOfReads fallback. Need to run a small set of test cases to ensure there aren't any new issues. I'm going to convert this to a draft PR until it is ready for review.

**Review by PedalheadPHX — APPROVED** [review]
No issues

**Review by PedalheadPHX — APPROVED** [review]
Ready to go

**Pull request merged**